pkg/osquery/table/kolide_launcher_autoupdate_config.go

```diff
@@ -0,0 +1,44 @@
+package table
+
+import (
+	"context"
+
+	"github.com/kolide/launcher/pkg/launcher"
+	"github.com/kolide/osquery-go/plugin/table"
+)
+
+const launcherAutoupdateConfigTableName = "kolide_launcher_autoupdate_config"
+
+func LauncherAutoupdateConfigTable(opts *launcher.Options) *table.Plugin {
```

Comment:
Why here  and not the existing `kolide_launcher_info` table? (I feel agnostic)

Reply:
The `kolide_launcher_info` table exposes information that was compiled into Launcher at build time. This table exposes configuration that was supplied at runtime. I think they are somewhat separate concerns.

In general, I'd prefer to keep tables smaller and more single purpose. Users can always join them if they want info from both.